scripts/src/javatests/org/oppia/android/scripts/todo/CommentCheckTest.kt

```diff
@@ -0,0 +1,94 @@
+package org.oppia.android.scripts.todo
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.testing.assertThrows
+
+/** Tests for [CommentCheck]. */
+class CommentCheckTest {
+  private val permalinkPrefix = "https://github.com/oppia/oppia-android/blob/"
+  private val dummySha1 = "51ab6a0341cfb86d95a387438fc993b5eb977b83"
+  private val dummySha2 = "74cd6a0341cfb86d95a387438fc993b5eb977b83"
+
+  @Rule
+  @JvmField
+  var tempFolder = TemporaryFolder()
+
+  /**
+   * This is the case when no comment is present on the issue. In this case the Github API results
+   * in an empty comment body.
+   */
+  @Test
+  fun testCommentCheck_emptyLatestCommentBody_commentCheckShouldPass() {
+    val latestCommentFile = tempFolder.newFile("latest_comment.txt")
+    val scriptFailureCommentFile = tempFolder.newFile("todo_list.txt")
+    val latestCommentContent = ""
+    val scriptFailureCommentContent =
+      """
+      The issue is reopened because of the following unresolved TODOs:
+      $permalinkPrefix$dummySha1/scripts/BUILD.bazel#L184
+      $permalinkPrefix$dummySha1/scripts/BUILD.bazel#L185
```

Comment:
Could we output these as a markdown list, instead, or does that mess up the automatic previewing that GitHub does?

Reply:
Ah! I tried and it looks like it messes up the automatic previewing.